ietf-document/draft-quic-4-sat/draft-kuhn-quic-4-sat-04.txt

```diff
@@ -0,0 +1,728 @@
+   networks could be in terms of the volume of acknowledgement traffic
+   (limited return path capacity) or in the number of acknowledgement
+   packets (e.g., when a radio-resource management system has to track
+   channel usage) or both.
+
+   TCP Performance Implications of Network Path Asymmetry [RFC3449]
+   describes a range of mechanisms that can mitigate the impact of path
+   asymmetry.  One simple method is to tell the remote endpoint to send
+   compound acknowledgments less frequently.  A rate of one ACK every
+   RTT/4 can significantly reduce this traffic.
+
+   Many of these mitigations have been deployed in satellite systems,
+   often as a mechanism within a PEP.  Despite their benefits over paths
+   with a high asymmetry of capacity, most mechanisms rely on being able
+   to inspect and/or modify the transport layer header information of
+   TCP ACK packets.  This is not possible when the transport layer
+   information is encrypted.  The QUIC transport specification may
+   evolve to allow the ACK Ratio to be adjusted.
+
+
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 9]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+6.  Discussion
+
+   Many of the issues identified already exist for any encrypted
+   transport service that uses a path that employs encryption at the IP
+   layer.  This includes endpoints that utilise IPsec at the network
+   layer, or use VPN technology over the satellite network segment.
+   These uses are unable to benefit from enhancement within the
+   satellite network segment, and often the user is unaware of the
+   presence of the satellite link on their path, except through
+   observing the impact it has on the performance they experience.
+
+   One solution would be to provide PEP functions at the termination of
+   the security association (e.g., in a VPN client).  Another solution
+   could be to fall-back to using TCP (possibly with TLS or similar
+   methods being used on the transport payload).  A final solution could
+   be to deploy and maintain a bespoke protocol tailored to high BDP
+   environments.  In the future, we anticipate that fall-back will
+   become less desirable, and methods that rely upon bespoke
+   configurations or protocols will be unattractive.  In parallel, new
+   methods such as QUIC will become widely deployed.  The opportunity
+   therefore exists to ensure that the new generation of protocols offer
+   acceptable performance over high BDP paths without requiring
+   operating tuning or specific updates by users.
+
+7.  Acknowledgements
+
+   The authors would like to thank Christian Huitema, Igor Lubashev,
+   Alexandre Ferrieux, Francois Michel, Emmanuel Lochin and the
+   partipants of the IETF106 side-meeting on QUIC for high BDP for their
+   useful feedbacks.
+
+8.  Contributors
+
+   TBD
+
+9.  IANA Considerations
+
+   TBD
```

Comment:
This document makes no request to IANA. This section could be removed prior to publication.